connector/connect/client/jvm/pom.xml

```diff
@@ -32,6 +32,7 @@
   <url>https://spark.apache.org/</url>
   <properties>
     <sbt.project.name>connect-client-jvm</sbt.project.name>
+    <guava.version>31.0.1-jre</guava.version>
```

Comment:
should we shade out the guava dependency here, to make sure it's breaking the client?